rfcs/0013-deprecate-currency-factory.md

```diff
@@ -0,0 +1,93 @@
+# Depreciation of Currency Factory - Assessment RFC
+
+This assessment aims to gather all relevant details and consequences of 
+depreciating Currency Factory and its replacements.
+
+## Reasoning
+
+There are several reasons we stand to benefit from replacing Currency Factory.
+These reasons are detailed below:
+
+### Currency Factory Ranges are Confusing
+  
+As detailed in [this](https://composablefinance.slack.com/archives/C031G5NT0CA/p1667492928188269) 
```

Comment:
please summarize details out of thread here

Reply:
(or you have done it already, just remove it or put into some no so distracting place - so we are not forced to read - i  know I do same way sometimes, but IMHO rfc could be cleaner)